src/ontology/run.sh.conf

```diff
@@ -1,3 +1,3 @@
 ODK_IMAGE=odkfull
-ODK_TAG=dev
+ODK_TAG=v1.5.1
```

Comment:
Is this supposed to still be in draft?

I thought maybe this was because a build hasn't been run yet, but the "QC" checkbox is checked, indicating that it has already run. But I don't see any PRs for such a build?

Reply:
I had a failure that is oak related on another pipeline so we should make sure that 151 works first

Reply:
Actually, it looks like you already ran it?
This says it's for #594, but it uses 1.5.1:
- #604

Reply:
@matentzn who is "we" here :) Are you running a feature branch build for this change?

Reply:
> @matentzn who is "we" here :) Are you running a feature branch build for this change?

Sorry I cant at the moment - I can do it next week, or one of you does it if you have cycles free. My laptop is running other builds at the moment relatively non-stop. I can do it Monday

Reply:
@twhetzel Leaving this up to you if you think we should run the build for this, and who should do that if so.

Reply:
```suggestion
ODK_TAG=v1.5.2
```